modules/mailer.js

```diff
@@ -31,7 +32,7 @@ async function sendReminders(classInfo, clientEmails){
                 from: '"OG YOGA" ' + process.env.SENDER_EMAIL,
                 to: process.env.SENDER_EMAIL,
                 bcc: clientEmails,
-                subject: TODO,
+                subject: 'OG Yoga: Your class ' + classInfo.class_name + 'starts in 30 minutes.',
```

Comment:
I don't think we should have a constant time of 30 minutes here as each class will start somewhere within the range of 15-30 minutes.

Reply:
Changed to "starting soon!"